### PR TITLE
[bitnami/*] Move utilities to support folder

### PR DESCRIPTION
**Pull request by superaleks**
**Description of the change**

Since Cypress will execute all files in the `integration` folder, there can be an empty segment in the reports. 
As an improvement task, we agreed to move all files which are not the test suite itself to the `support` folder. 

Taking a look at the [following example](https://github.com/bitnami/charts/pull/11146) we can apply the same across all covered assets.

### Checklist

- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in the agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Review by dani8art — DISMISSED** [review]

File: .vib/spring-cloud-dataflow/cypress/cypress/support/spring_cloud_dataflow_spec.js [hunk 1]
```diff
@@ -1,6 +1,9 @@
 /// <reference types="cypress" />
```
Comment (joancafom) [inline .vib/spring-cloud-dataflow/cypress/cypress/support/spring_cloud_dataflow_spec.js:1]: As said, I reckon this file belongs to the `integration` folder.

**Review by joancafom — CHANGES_REQUESTED** [review]
In addition to `spring-cloud-dataflow`, I think you are missing changes in the `suitecrm_spec.js` file to indicate the new location of `utils.js`.

File: .vib/spring-cloud-dataflow/cypress/cypress/support/spring_cloud_dataflow_spec.js (hunk 1 above)
Comment (superaleks) [reply .vib/spring-cloud-dataflow/cypress/cypress/support/spring_cloud_dataflow_spec.js]: Oops, great catch. I fixed this in both instances.

**Review by joancafom — APPROVED** [review]
LGTM!

**Review by dani8art — APPROVED** [review]

**Pull request merged**